Base/QTCore/qSlicerCoreIOManager.cxx

```diff
@@ -482,10 +482,22 @@ QString qSlicerCoreIOManager::forceFileNameValidCharacters(const QString& filena
 
   // Remove leading and trailing spaces
   sanitizedFilename = sanitizedFilename.trimmed();
-
   return sanitizedFilename;
 }
 
+//-----------------------------------------------------------------------------
+QString qSlicerCoreIOManager::forceFileNameMaxLength(const QString& filename, int maxLength/*=25*/)
+{
+  QString extension = extractKnownExtension(filename, nullptr);
```

Comment:
add `this->` or `qSlicerCoreIOManager::` to make it more clear where the function comes from